TIP-0003/README.md

```diff
@@ -0,0 +1,273 @@
+  - The policy $p$ is attached to the transaction.
+  - The number of group constructor tokens with identifier $g$ in the output of the transaction is strictly bigger than 0.
+  - The registration UTXO referenced in $p$ is present in the inputs and contains LVLs.
+
+#### Minting of Series Constructor Tokens
+
+The minting of a series constructor token requires to burn a certain amount of LVLs and to provide te policy that describes the series. This requires the submission of a minting transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction:
+
+- *Check Moving Series Tokens:* Let $s$ be a series identifier, then the number of Series Constructor Tokens with group identifier $s$ in the input is equal to the number of the number of Series Constructor Tokens with identifier $s$ in the output.
+- *Check Minting Constructor Tokens:* Let $s$ be a series identifier and $p$ the series policy whose digest is equal to $s$, all of the following statements are true:
+  - The policy $p$ is attached to the transaction.
+  - The number of series constructor tokens with identifier $s$ in the output of the transaction is strictly bigger than 0.
+  - The registration UTXO referenced in $p$ is present in the inputs and contains LVLs.
+
+#### Minting of an Asset Token
+
+The minting of an asset requires to use one group and one series constructor token. This requires the submission of a minting transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction.
+
+- *Check Minting of Asset Tokens:* Let $A_0$ ... $A_{n-1}$ be the $n$ Asset Minting Statements to mint $m_0$, $m_1$, ..., $m_{n-1}$  tokens with identifier  ($g$, $s$) , all of the following statements are true:
+
+  - All Asset Minting Statements $A_0$ ... $A_{n-1}$  are attached to the transaction.
+
+  - Let $in$ be the number of assets with identifier ($g$ ,$s$) in the input and $out$  number of assets with identifier ($g$ ,$s$) in the output, then: 
+    $$
+    in + \sum_{i = 1}^{n - 1} m_i = out
+    $$
+
+  - For all $A_i$, all UTXOs referenced are unique.
+
+  - For all $A_i$, the token supply specified in the referenced series is equal to the quantity attribute in $A_i$.
+
+  - Let $sin$ be the total number of series constructor tokens with identifier $s$ in the input, $sout$ the total number of series constructor tokens with identifier $s$ in the output, and $burned$ the number of $A_i$ where the referenced series specifies a token supply, then we have:
+
+    $$sin - burned = sout$$
+
+### Transacting with Assets
+
+Transaction with assets are performed in the same way as transactions with LVLs. The only difference is that assets allow for different degrees of fungibility. At a chain level two assets are fungible if they can both be stored in the same UTXO. This poses a problem when we deal with two assets that are fungible at one level but not at the other. 

```

Comment:
`At a chain level two` should probably be `At a chain level, two` just to avoid the accidental definition of "level two assets" which is how I read it on the first pass 😆